client-apis/encrypt.md

```diff
@@ -153,6 +170,10 @@ Each frame of the [message body](../data-format/message-body.md) is serialized w
   - The plaintext contains part of the input [plaintext](#plaintext) this frame is encrypting.
 - [Authentication Tag](../data-format/message-body.md#authentication-tag): MUST be the authentication tag outputted by the above encryption.
 
+If [Plaintext Length Bound](#plaintext-length-bound) was specified on input
+and this operation can determine that the plaintext being encrypted has a length greater than this value,
```

Comment:
```suggestion
and this operation determines that the plaintext being encrypted has a length greater than this value,
```
How could the operation not determine this?
This also resolves my "top billing comment" :)